src/DocblockFieldValidator.php

```diff
@@ -6,7 +6,7 @@
 
 class DocblockFieldValidator extends FieldValidator
 {
-    public const DOCBLOCK_REGEX = '/@var ((?:(?:[\w?|\\\\<>])+(?:\[])?)+)/';
+    public const DOCBLOCK_REGEX = '/@var ((?:(?:[\w?|\\\\<>,\s])+(?:\[])?)+)/';
```

Comment:
Personally this exceeds my capability to quickly understand the regex, no matter how good I'm with them 😄

Reply:
I did my best to add some docs :D